active/package-maintainers.md

```diff
@@ -0,0 +1,126 @@
+# Package Maintainers Working Group
+
+## Scope of responsibilities
+
+This group supports package maintainers in the Django ecosystem.
+Packages are an important part of the experience of using Django, and their maintainers can benefit from different forms of support than Django contributors and users.
+
+The goals of the group are:
+
+- To facilitate discussions and share knowledge about common issues or tasks in package maintenance.
+- To coordinate efforts to improve the health and quality of packages in the Django ecosystem.
+- To advocate for the needs of package maintainers in the Django community.
+
+This is a broad definition of the group’s remit.
+Current members select specific activities to undertake based on their interests and availability.
+See [Group activities](#group-activities) for examples of what the group may choose to do.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the group operates with:
+
+- No special powers delegated by the board to the group.
+- No specific "Django Software Foundation" actions the group can take directly.
+
+Any request for funding or resources are made via existing processes outside of this group.
+This can be revised in the future as and when the group identifies specific recurring needs.
+
+## Membership
+
+- Chair: You?
+- Co-Chair: You?
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Thibaud Colas
+- Other members:
+  - You?
+
+## Eligibility
+
+Membership is open to all Individual Members of the Django Software Foundation.
```

Comment:
About this membership requirement,

The only other WG I could find with an individual membership requirement is [Social Media WG](https://github.com/django/dsf-working-groups/blob/main/active/social-media.md). For that case, I think it makes because of the particular working topic, but I don't think it should be a requirement here.

[DSF Individual Memberships page](https://www.djangoproject.com/foundation/individual-members/) says

> Individual Members are appointed by the DSF in recognition of their contribution to the DSF's mission of advancing and promoting Django, protecting the framework's long-term viability, and advancing the state of the art in web development.
>
> Contribution to the DSF's mission takes many forms. Here are some non-exhaustive examples of the categories of work that might qualify:
> ...
> * Serving on a [DSF Working Group](https://github.com/django/dsf-working-groups).

If serving on a WG is a soft requirement for a membership, having that condition reversed sounds counterintuitive.

The more inclusive requirement would be, at least one of:
* Having open source contribution/maintenance experience
* Having team lead/management experience
* Having enough professional experience

Reply:
I feel open source maintainership experience is very important to contribute productively to a discussion about package maintainer best practices.  There are things you will experience as a maintainer and release manager that you will never experience as solely a contributor.  Maybe that should be a requirement for chair/co-chair?

Reply:
It depends. Does the Chair/co-chair have some extra ability associated with the role? Or are they the people managing the logistics of the group? If it's the latter, I think the requirement is anyone who wants to do the job. If it's the former, I'm on the fence. Well as long as the group has people in it with that experience :grin:

Reply:
@tim-schilling I see your point. I was assuming the chair/co-chairs would have some sort of responsibility to resolve tie votes on recommendations or something of the sort that would require technical expertise. So maybe it makes sense to clarify the role and responsibilities of the chairs?